Components/Combat/DefaultTargetingProvider.cs

```diff
@@ -261,7 +261,7 @@ public bool IsInRange(Vector3 position, TrinityPower power)
             if (Core.Player.IsInBossEncounter && currentTarget != null && currentTarget.ActorSnoId == (int) SNOActor.Belial)
                 return distance <= rangeRequired;
 
-            return distance <= rangeRequired && (distance < 7f || IsInLineOfSight(position));
+            return distance <= rangeRequired && (distance <= 7.0f || IsInLineOfSight(position));

```

Comment:
This might cause issues for monsters behind the walls and such as we are not checking LoS for actors closer then 7 yards now. Just leaving this as a note here.